tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -1337,4 +1337,13 @@ class WskBasicUsageTests
             }
             tmpProps.delete()
     }
+
+    behavior of "Wsk activation"
+
+    it should "reject activation list with a limit that exceeds the maximum limit" in {
+        val maxLimit = 200
```

Comment:
The max limit is hardcoded in open/core/controller/src/main/scala/whisk/core/controller/Activations.scala. `val cappedLimit = if (limit == 0 || limit > 200) 200 else limit`